src/Agent.Listener/CommandSettings.cs

```diff
@@ -281,7 +262,19 @@ public string GetPool()
 
         public string GetToken()
         {
+            string value = null;
+
+            if (Configure != null)
```

Comment:
nit: This pattern shows up a lot. I wonder if Configure and Remove should have a base command class that has all the shared stuff. Then you could say something like `GetConfigureOrRemove()?.Token` here.

Reply:
I think that's a better way to do it.  I'll update!